src/game_api/entities_chars.hpp

```diff
@@ -35,14 +36,15 @@ class Ai
     int32_t timer;
     /// AI state (patrol, sleep, attack, aggro...)
     int8_t state; // 8 = HH agro
-    int8_t unknown12;
+    int8_t last_state;
     /// Levels completed with, 0..3
     uint8_t trust;
     /// How many times master has violated us
```

Comment:
I just saw this on here. Can we please not use the word master.

Reply:
wdym?

Reply:
Oh no bad word maybe slavery will come back if you repeat it three times. I don't care, but you should probably remove strings00 and the sound system too.

Reply:
If we have previously decided to censor the word damsel then I think not using master is just natural  ;)

Reply:
Your reference to the sound system and strings00.str is excessively childish. The usage there is in a clearly different context than what is used here. If you can't see that I am seriously sorry for you.

Reply:
This is all a joke to me like the thing I wrote in the docs a long time ago, cause the game is clearly joking on the subject too. Robots waiting for their old masters in the journal is indeed different vibe. This just sounds like virtue signalling. I guess all the human sacrifice and murder in the game is fine though, as long as some arbitrary words are cancelled every year. Do what you want, I don't care.

In my mind we changed damsel because the game calls them pets btw, nothing to do with censorship.

Reply:
It's an enigma to me who I would be virtue signaling to. Did you expect me to put this into the changelog or something?

But whatever, I will indeed do this somewhere else if I want to. Just because doing what I think is right publicly suddenly makes it bad I won't stop doing it.